docs/lib/datastore/fragments/flutter/getting-started/50_initDataStore.md

```diff
@@ -16,28 +14,38 @@ Your code should look like this:
 ```dart
 import 'package:amplify_flutter/amplify.dart';
 import 'package:amplify_datastore/amplify_datastore.dart';
-import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+import 'package:flutter/widgets.dart';
 
 import 'amplifyconfiguration.dart';
 import 'models/ModelProvider.dart';
 
 class MyAmplifyApp extends StatefulWidget {
+  @override
+  _MyAmplifyAppState createState() => _MyAmplifyAppState();
+}
 
-    @override
-    void initState() {
-        super.initState();
-        _configureAmplify();
+class _MyAmplifyAppState extends State<MyAmplifyApp> {
+  @override
+  void initState() {
+    super.initState();
+    _configureAmplify();
+  }
+
+  void _configureAmplify() async {
+    // Add the following line to add DataStore plugin to your app
+    Amplify.addPlugin(AmplifyDataStore(modelProvider: ModelProvider.instance));
```

Comment:
Hm, maybe you were just trying to be consistent - though maybe the more verbose code would be better for documentation.

I am a little confused why the plugin has to be added twice though, any ideas?

Reply:
Yeah, I was just trying to be consistent. I don't have a strong preference between the two as long as they are consistent.

It doesn't have to be added twice. The first snippet is just the minimal code that someone would add if they were integrating Datastore into an app that already uses amplify, and the second is the full example of what your app would look like if you were integrating it into a new app.